state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1351,14 +1400,20 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 						Self::deposit_event(Event::DepositBoosted {
 							deposit_address: deposit_address.clone(),
 							asset,
-							amount,
+							amounts: used_pools,
+							channel_id,
 							deposit_details: deposit_details.clone(),
 							ingress_fee,
 							boost_fee: boost_fee_amount,
 							action,
 						});
 					},
 					Err(err) => {
+						Self::deposit_event(Event::InsufficientBoostLiquidity {
+							boost_id: id,
```

Comment:
nit, but this makes it a bit easier to follow
```suggestion
							boost_id: prewitnessed_deposit_id,
```